ONSdigital/ssdc-rm-support-tool
# 2713 sms fulfilment templates and requests

# Motivation and Context

After the recent notify sms fulfilment changes that went in, we needed to add the request an sms fulfilment feature to the support tool.

# What has changed



- Able to create an SMS template
- Request a sms fulfilment on a case
# How to test?



- Run with other branches on PR
- Create an sms template for a uac `["__uac__"]` and another one for non uac `[]`
- Enable a survey to use the template
- Request sms fulfilments on a case. The one template should generate a uac and the other should just have an event against it

# Links



[Trello](https://trello.com/c/8i3NpABx/)
